papers/paper_bench_2025.md

```diff
@@ -0,0 +1,179 @@
+---
+author: Martiño Ríos García
+date: 2025-04-16
+title-block-banner: paper_bench_images/judges.png
+bibliography: ../references.bib
+---
+
+# PaperBench: Evaluating AI’s Ability to Replicate AI Research
+
+## Why discussing this paper?
+
+I chose PaperBench from OpenAI [@starace2025paperbench0] because:
+
+- It is a recent paper that proposes a new benchmark for evaluating LLM-powered agents that work in developing AI research.
+- Specifically, the task of reproducing existing work should be the first milestone in the development of LLM-powered agents that can autonomously conduct research.  
+- They use LLM-as-Judge for evaluating some of the tasks.
+
+## Context
+
+Agents are becoming increasingly capable of performing complex tasks, including some sparks of autonomous independent research [@lu2024ai]. However, perhaps one of the first steps in AI and coding research is the ability to reproduce existing work. This is a crucial step in validating and building upon previous research findings. While there are many benchmarks for evaluating LLMs in various domains, there are few that focus specifically on the task of reproducing AI research, and more specifically, only having access to the paper.
+
+## Prior Work
+
+### SciCode: A Research Coding Benchmark Curated by Scientists
```

Comment:
perhaps you could almost move some of those notes to the Chemical Reviews ... I feel your review here is more elaborate than what we have right now on agent (evals)